tests/e2e/cucumber/features/integrations/share.ocis.feature

```diff
@@ -1,24 +1,21 @@
 Feature: share
 
   Background:
-    And "Admin" disables share auto accepting
-
-  Scenario: folder
-    Given "Admin" creates following users
+    Given "Admin" disables share auto accepting
```

Comment:
Maybe this can be removed if this feature only runs on ocis? For ocis this is default isn't it?